index.html

```diff
@@ -18,11 +18,75 @@
 <body>
   <header>
     <div class="container">
+      <div class="row">
+        <h1 class="journal-header">Code Journal</h1>
+      </div>
     </div>
   </header>
   <main class="container">
-    <div data-view="entry-form"></div>
-    <div data-view="entries" class="hidden"></div>
+    <div data-view="entry-form">
+      <form action="#" class="form">
+
+        <div class="row">
+          <!-- form title -->
+          <h2 class="page-title">New Entry</h2>
+        </div>
+
+
+        <div class="row">
+          <!-- Image -->
+          <div class="column-half">
+            <img src="images/placeholder-image-square.jpg" alt="Placeholder Image" class="entry-image">
+          </div>
+
+          <div class="column-half">
+            <!-- title -->
+            <div class="label-input-div">
+              <div class="label-div">
+                <label for="title">Title</label>
+              </div>
+              <div class="input-div">
+                <input type="text" class="input-title" id="title" name="title" required="required">
+              </div>
+            </div>
+
+            <!-- PhotoURL -->
+            <div class="label-input-div">
+              <div class="label-div">
+                <label for="photo-url">Photo URL</label>
+              </div>
+              <div class="input-div">
+                <input type="url" name="photoURL" id="photoURL" class="input-photo-url" required="required">
```

Comment:
The `id` for this `input` should be `photo-url` to match the `for` attribute of your `label` on line 56

Reply:
I address this issue by changing the `for` attribute of the label to photoURL so I do not need to change the JavaScript query selection.